client/packages/inventory/src/Stocktake/DetailView/Toolbar.tsx

```diff
@@ -77,6 +87,15 @@ export const Toolbar: FC = () => {
             }}
             debounceTime={0}
           />
+          <Box sx={{ marginRight: 2 }}>
+            <Switch
+              label={t('label.group-by-item')}
```

Comment:
side note: `"label.group-by-item": "Group by Item",` is in three files now, and could move to `common.json`

Reply:
Cool - done e5491f58e